src/index.html

```diff
@@ -32,20 +32,48 @@ <h1 class="heading">CROP AI</h1>
           <a href="#" class="button-Index" id="about_btn">ABOUT</a>
           <a href="#" class="button-Index" onclick="show_input_container()">ANALYZE</a>
           <div id="about_content">This is a crop advisory app for farmers and gardeners. It can diagnose pest damage, plant disease and nutrient deficiencies that are affecting crops and can offer corresponding treatment measures.</div>
-         
+
         </div>
       </div>
 
       <div id="input_container" class="hidden"></div>
-      
-      <div id="result_container" class="hidden"></div>
-    
+
+      <div id="result_container">
+        <div id="box">
+          <div id="float-child">
+            <img id= "pepper-bell" src="./img/leaf.jfif" alt="Image not found"  height= 300px width="200px">
+          </div>
+          <div id="float-child">
+            <div id="heading">
+              <h3>Pepper Bell Bacterial Spots </h3>
+            </div>
+            <p id="list_head">Symptoms</p>
```

Comment:
id should be unique and should be used for one element, you want use class (list_head, symptoms)
change symptoms to a more meaningful class like list_content